packages/@okta/vuepress-site/docs/guides/sharing-cert/next-steps/index.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Next steps
+---
+You should now understand how to share application key credentials between apps. If you got stuck, post a question in our [Developer Forum](https://devforum.okta.com).
```

Comment:
The line "If you got stuck, post a question in our [Developer Forum]" is standard text in our next steps - but it occurs to me that we should put this support statement on the first page of our guides instead of at the end - because if they get stuck, they might not get to this last page. 
ie. "If at any point you get stuck, post a question in our ...."
We can discuss this at a team level if you want to ignore this comment for now. :-)

Reply:
let's discuss it with the team and then make the change, if needed, for all.